rotkehlchen/inquirer.py

```diff
@@ -648,7 +672,17 @@ def _find_usd_price(
                 price, oracle = Inquirer._query_fiat_pair(base=asset, quote=Inquirer.usd)
                 return price, oracle, False
 
-        # continue, asset isn't fiat or a price can be found by one of the oracles (CC for example)
+        # continue, asset isn't fiat, check manual prices
+        if (price_result := Inquirer._try_oracle_price_query(
```

Comment:
Hmm okay so you now run manual oracle before of o red it correctly ... But the manual oracle is reran in the for loop no? Didn't we say it should be removed during stand up?